### DISCUSSION TITLE
The election is ongoing

### DISCUSSION THREAD

**Discussion by ryjones** [LF-Decentralized-Trust/governance#64] [Announcements]
If you [are eligible to vote](https://lf-decentralized-trust.github.io/tac-eligibility-check/) please visit [this link](https://vote.heliosvoting.org/helios/e/2025-LF-Decentralized-Trust-TAC-election) while logged in to GitHub and vote. Voting ends [14 Nov 2024](https://lf-decentralized-trust.github.io/governance/member-info/election-timeline.html). The top six will be on the TAC; the board will then choose five people to fill the remaining seats.

<img width="442" alt="Screenshot 2024-11-08 at 08 52 52" src="https://github.com/user-attachments/assets/ce4166ad-99f2-4be1-91a1-ff1998f0443a">

**Comment by ryjones** [comment]
@0xivanov @2byrds @acuarica @adaagava @adityajoshi12 @adrianbatuto @agadzhalov @ahamlat @ahmed82 @ajsutton @akarabashov @aklenik @aldousalvarez @ale-linux @alexandratran @alexandrosfilios @alex-kuzmin-hg @alex-semenyuk @alexstroke1 @AlfredoG87 please vote.

**Reply by ryjones** [reply]
@alinelermen @AliNik4n @amagyar-iohk @amanji @anastasiya-kovaliova @AndreAugusto11 @andremacedopv @andrewb1269hg @andrew-coleman @andrewwhitehead @annamariyaivanova @anthony-swirldslabs @antonydenyer @anusha-padmanabhan @anwalker293 @ArchanaArige @arianejasuwienas @arndey @arner @artemananiev see above

**Reply by ryjones** [reply]
@Artemkaaas @ArturoGarciaRegueiro @a-saksena @asararatnakar @ashcherbakov @Ashe-Oro @atala-dev @ata-nas @atoulme @attest08 @auer-martin @awrichar @a-zorina @BartoszSolkaArianelabs @BAStos525 @baziorek @beeradb @benjsmi @berendsliedrecht @bestbeforetoday see above

**Reply by ryjones** [reply]
@bgravenorst @bibitibooo1 @bilyana-gospodinova @blockchaintimothy @bluele @bryce-mcmath @bsandmann @bstasyszyn @burdettadam @bvavala @bvoiturier @C0rWin @c2bo @canton-network-da @cdivitotawela @celder628 @cendhu @Chengxuan @chookly314 @chumbert

**Reply by ryjones** [reply]
@cjhowland @ckpaliwal @cloudspores @cmickeyb @coeniebeyers @conanoc @conor10 @criro1 @cristianIOHK @CryptoKnightIOG @cryptphil @curtis-h @cvarjao @cweagans @Dale-iohk @dalvizu @danielksan81 @daniellehrner @david-bakin-sl @davidkel

**Reply by ryjones** [reply]
@thiagoromanos @thomas-swirlds-labs @timofeevmd @TimoGlastra @tinker-michaelj @tinnendo @tock-ibm @Toktar @tolorukooba @tomzhenghedera @travismccracken11 @ty-swirldslabs @Udhayakumari @urgetolearn @urosmrvic-iohk @usmansaleem @VadneyK @varshapichandi30 @victor-yanev @vitiko @vtronkov @WadeBarnes @wadeking98 @waynecollier-da @web3-nomad @womfoo @WRRicht3r @xermicus @xiaor2 @xichen1 @xin-hedera @xprazak2 @yacovm @yamkovoy @yeasy @yiliev0 @yosiftouma @YoungHypo @yshyn-iohk @yukiknd @yungu23 @zhuyuanmao @zondervancalvez

**Reply by ryjones** [reply]
@PatStLouis @peacekeeper @peterbroadhurst @petermunnings @petevielhaber @petreze @Petyo-Lukanov @pfi79 @pinges @popowycz @poulok @povolev15 @prakashngit @quiet-node @r3domfox @RafaelAPB @RamilMus @RAMTO @RaphaelMessian @ray-roestenburg-da @reachtobhavesh @Reccetech @RickyLB @rmbrt @rsinha @rustyShacklefurd @ruzell22 @rwalworth @rwat17 @s7santosh @s8sato @sadityakumar9211 @safinsaf @saikumarbommakanti @salaheldinsoliman @SamHSmith @SamMayWork @samswag @samuelvenzi @SamYuan1990
@sandeepnRES @san-est @satota2 @saurabhkumarkardam @se7enarianelabs @seanyoung @sergmetelin @shankars99 @shemnon @Sheng-Long

**Reply by ryjones** [reply]
@MuthuSundaravadivel @mversic @nadezhdapopovaa @nain-F49FF806 @Nana-EC @nano-adhara @natanasow @nathanklick @Neeharika-Sompalli @netopyr @Neurone @nhoriguchi @NhoxxKienn @niall-shaw @nickeynikolovv @NickSneo @nidhi-singh02 @nirbosl @nistanimirov @nithin-pankaj
@nodlesh @non-fungible-nelson @norbert-kulus-arianelabs @ntsbs @nvzhu @nxsaken @ochikov @OlegMazurov @onelapahead @outoftardis @outSH @outsidethecode @pasquale95 @pathornteng @patlo-iog @Patrik-Stas @shoaebjindani @shorsher @shotexa @siburu
@siladu @SimiHunjan @smallestant @sownak @StanislawMazowieckiAriane @stefashkaa @steven-sheehy @stoqnkpL @stoyanov-st @Stukalov-A-M see above

**Reply by ryjones** [reply]
@survived @suvajit-sarkar @SvetBorislavov @svetoslav-nikol0v @svienot @swaptr @takemiyamakoto @takeutak @TanjinAlam @tannerjfco
@tareknaser @technomad01 @TelegramSam @thenswan @TheTechmage see above

**Reply by ryjones** [reply]
@davidkhala @dbluhm @DCNick3 @dechdev @denisandreenko @derektriley @devanshpurani7 @dexhunter @dhh1128 @diega @dikel @dima74 @dimaxgl @dinkar-jain @dkulic @dmitrivenger @dnakashima @dougvk @dr20240304 @DragonDev1906
@dshuffma-ibm @dviejokfs @dwertent @dzikowski @ebadiere @Echsecutor @ed-marquez @edward-swirldslabs @edwin-greene @eierina @ElijahLynn @elribonazo @ericleponner @erigara @essbante-io @esune @etschelp @Evdokia-Georgieva @EzequielPostan @fab-10
see above

**Reply by ryjones** [reply]
@FabioPinheiro @FayyaadhAdams @fengyangsy @ff137 @fowlerrr @fqutishat @g2flyer @gabriel-indik @Gabriel-Trintinalia @garyschulte @Gavinok @gcapuzzi @genaris @gfukushima @gmulhearn @goncalo-frade-iohk @gregscullard @gsstoykov @gtebrean @gv-timur see above

**Reply by ryjones** [reply]
@gynura @HagarMeir @Hejwo @ianco @ibankov @iliecirciumaru @iljabvh @imalygin @instamenta @isegall-da @IvanKavaldzhiev @ivaylogarnev-limechain @ivaylonikolov7 @Ivo-Yankov @iwsimon @IyNew-Letni @izik1 @izuru0 @JamesKEbert @jamshale see above

**Reply by ryjones** [reply]
@jasperpotts @JeffreyDallas @jeromy-cannon @jesusdiazvico @jflo @jframe @JivkoKelchev @jjohannes @jleach @jnels124 @joaniefromtheblock @joshuafernandes @jovfer @Jserrano27 @jsync-swirlds @jsyro @jt-nti @kacper-koza-arianelabs @karimstekelenburg @KElkhiyaoui see above

**Reply by ryjones** [reply]
@kenthejr @ketulsha @kfa-aguda @kimbor @kingnhcomcast @Kirqkas @knagware9 @konstantinabl @kselveliev @kukgini @leninmehedy @litt3 @lpetrovic05 @lu-pinto @lucassaldanha @LucasSte @lukelee-sl @lynnbendixsen @m-courtin @m4sterbunny see above

**Reply by ryjones** [reply]
@macfarla @manish-sethi @manishdas12 @manoranjith @marcelamelara @Mark-Swirlds @Matilda-Clerke @matkt @matteriben @mattklepp @mattp-swirldslabs @maurges @mayrosenbaum @mbaxter @mbrandenburger @mbwhite @mepeltier @mgarbs @mgCepeda @mgmgwi see above

**Reply by ryjones** [reply]
@mgoelswirlds @mhess-swl @michalrozekariane @michielmulders @mikaellafs @mikelodder7 @milosbackonja @milosh86 @mineme0110 @Mingela @mirgee @MiroslavGatsanoga @mkbreuningIOHK @mkrielza @mmalik-al @modemn @monar24 @moritzkiefer-da @moscd3 @munapower see above

**Comment by nain-F49FF806** [comment]
Gentle reminder to **_NOT_** enter your Github password directly on their site! 

Only _after_ you click the login button at the bottom, the actual Github OAuth page will open.

---

p.s: If you did accidentally enter your github credentials there, imo it would be wise to change your password. 

p.p.s: I have created an issue upstream regarding the confusing UI. https://github.com/benadida/helios-server/issues/398
If you have any input, do consider sharing.

**Reply by ryjones** [reply]
@nain-F49FF806 thank you!
